nova/objects/compute_node.py

```diff
@@ -330,6 +332,10 @@ def save(self, prune_stats=False):
 
         updates = self.obj_get_changes()
         updates.pop('id', None)
+        extra_resources = updates.pop('extra_resources', None)
```

Comment:
Please add a method similar to the following methods to pop the extra_resources.
         self._convert_stats_to_db_format(updates)
         self._convert_host_ip_to_db_format(updates)
         self._convert_supported_instances_to_db_format(updates)